### PR TITLE
FACT-1212 changed users tab to go to idam user dashboard

### PR DESCRIPTION
**Pull request by justiceia**
### JIRA link ###
https://tools.hmcts.net/jira/browse/FACT-1212

### Change description ###
removed all functionality surrounding the users link within fact admin. This now redirects to idam user dashboard.
added prod line into cnp-flux-config > https://github.com/hmcts/cnp-flux-config/pull/16901

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: charts/fact-admin/values.yaml [hunk 1]
```diff
@@ -26,5 +26,6 @@ nodejs:
     REDIS_HOST: "fact-admin-session-storage-{{ .Values.global.environment }}.redis.cache.windows.net"
     FRONTEND_URL: "https://fact.{{ .Values.global.environment }}.platform.hmcts.net"
     IMAGE_BASE_URL: "https://fact{{ .Values.global.environment }}.blob.core.windows.net/images"
+    IDAM_USER_DASHBOARD: "https://idam-user-dashboard.{{ .Values.global.environment }}.platform.hmcts.net"
```
Comment (amstevenson) [inline charts/fact-admin/values.yaml:29]: We probably need to remove the other variables that won't be used after this is merged in 

For example: IDAM_ADD_USER_URL won't be used afterwards

Comment (amstevenson) [reply charts/fact-admin/values.yaml:29]: IDAM_UPDATE_USER_URL, IDAM_ADD_USER_URL (think its just those two)

File: config/custom-environment-variables.yaml [hunk 2]
```diff
@@ -12,6 +12,7 @@ services:
     clientID: OAUTH_CLIENT_ID
     clientSecret: OAUTH_CLIENT_SECRET
     callbackURL: OAUTH_CLIENT_REDIRECT
+    userDashboard : IDAM_USER_DASHBOARD
```
Comment (amstevenson) [inline config/custom-environment-variables.yaml:15]: I'd change variable name to userDashboardURL to be clearer (i.e to denote what the string is)

File: charts/fact-admin/values.yaml (hunk 1 above)
Comment (amstevenson) [reply charts/fact-admin/values.yaml:29]: I'd also change the variable to be IDAM_USER_DASHBOARD_URL to denote the purpose of it and to follow the same structure as the other variable names above

File: src/main/app/controller/users/UserController.ts [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-
+import config from 'config';
 
 export class UserController {
 
   public async get(req: AuthedRequest, res: Response): Promise<void> {
-
-    res.render('users/index');
+    const userDashboard: string = config.get('services.idam.userDashboard');
+    res.redirect(userDashboard);
```
Comment (amstevenson) [inline src/main/app/controller/users/UserController.ts:9]: If you want you can shorten this to one line: 

res.redirect(config.get('services.idam.userDashboard')); 

it should be bringing it across as a string based on the type its created as in the config file

Comment (justiceia) [reply src/main/app/controller/users/UserController.ts]: fixed

File: config/custom-environment-variables.yaml (hunk 2 above)
Comment (justiceia) [reply config/custom-environment-variables.yaml]: fixed

File: charts/fact-admin/values.yaml (hunk 1 above)
Comment (justiceia) [reply charts/fact-admin/values.yaml]: fixed, added cnp flux config line here https://github.com/hmcts/cnp-flux-config/pull/16901

**Review by amstevenson — APPROVED** [review]
